jurigged/codetools.py

```diff
@@ -802,6 +850,51 @@ def reevaluate(self, new_node, glb):
         self._codeobj = new_obj.__code__
         return new_obj
 
+    def reevaluate(self, new_node, glb):
+        # patch: The assertion rewrite is from pytest. Jurigged doesn't
+        #              seem to have a way to add rewrite hooks
+        # TODO: We should fix this in jurigged itself, but porting over from here for now:
+        # https://github.com/breuleux/jurigged/issues/29
+        new_node = self.apply_assertion_rewrite(new_node, glb)
```

Comment:
consider moving the CONFIG check here so that the `apply_assertion_rewrite` logic is simpler. You wouldn't expect a function with that name to not do what it says.

Reply:
This comment is obsolete as after having time to stew on this, I don't think jurigged should own fixing pytest, it just needs to provide a means to do so without monkey patching the library.